hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/distributions/state/ManageDistUIState.java

```diff
@@ -101,26 +102,23 @@ public ManageDistFilters getManageDistFilters() {
      * @return the slectedDistributions
      */
     public Optional<Set<DistributionSetIdName>> getSelectedDistributions() {
-        return selectedDistributions == null ? Optional.empty() : Optional.of(selectedDistributions);
+        return Optional.ofNullable(selectedDistributions);
     }
 
     /**
      * @return the lastSelectedDistribution
      */
     public Optional<DistributionSetIdName> getLastSelectedDistribution() {
-        return lastSelectedDistribution == null ? Optional.empty() : Optional.of(lastSelectedDistribution);
+        return Optional.ofNullable(lastSelectedDistribution);
     }
 
-    /**
-     * @param lastSelectedDistribution
-     *            the lastSelectedDistribution to set
-     */
-    public void setLastSelectedDistribution(final DistributionSetIdName lastSelectedDistribution) {
-        this.lastSelectedDistribution = lastSelectedDistribution;
+    @Override
+    public void setLastSelectedEntity(final DistributionSetIdName value) {
+        this.lastSelectedDistribution = value;
     }
 
-    public void setSelectedDistributions(final Set<DistributionSetIdName> slectedDistributions) {
-        selectedDistributions = slectedDistributions;
+    public void setSelectedEnitities(final java.util.Set<DistributionSetIdName> values) {
```

Comment:
Remove the full qualified package from method parameter. Set is aleady imported in this class.